cluster/router/router.go

```diff
@@ -50,3 +55,39 @@ type PriorityRouter interface {
 	// 0 to ^int(0) is better
 	Priority() int64
 }
+
+// Poolable caches address pool and address metadata for a router instance which will be used later in Router's Route.
+type Poolable interface {
```

Comment:
this interface name is not a noun. So it should be 'Pool' or "RouterMetaPool'.

Reply:
I still believe `Poolable` is a better interface name :). It is a common practice in Java, and even in dubbo go there's another interface named `Listenable`

Reply:
got it.